Library/Homebrew/language/node.rb

```diff
@@ -0,0 +1,35 @@
+module Language
+  module Node
+    def self.npm_cache_config
+      "cache=#{HOMEBREW_CACHE}/npm_cache\n"
+    end
+
+    def self.setup_npm_environment
+      npmrc = Pathname.new("#{ENV["HOME"]}/.npmrc")
+      # only run setup_npm_environment once per formula
+      return if npmrc.exist?
```

Comment:
Oh, heh. This is what happens when I'm having a fairly long day; my brain ceases to function on anything above a "Keep Dom Alive" capacity. I briefly forgot we set it to `.brew_home` rather than just the `buildpath`. ` ¯\_(ツ)_/¯`